src/Marten/ISerializer.cs

```diff
@@ -165,6 +165,8 @@ public enum NonPublicMembersStorage
     {
         Default = 0,
         NonPublicSetters = 1,
-        NonPublicDefaultConstructor = 2
+        NonPublicDefaultConstructor = 2,
+        NonPublicConstructor = 4,
```

Comment:
Minor thing, shouldn't the value be `3` rather than `4`?

Reply:
I'm adding that to allow bit flag usage (e.g. `NonPublicMembersStorage.NonPublicSetters | NonPublicMembersStorage.NonPublicDefaultConstructor`). It needs to be 2^n

Reply:
See also: https://github.com/JasperFx/marten/pull/2004/files#diff-f899e2a4569989d2b4fbf225960c0a07bba52c31d6b434b964ffb02dfa4eb148R33

Reply:
Okay, makes sense 👍